src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -104,6 +104,9 @@
   <ItemGroup>
     <NuGetPackageToIncludeInVsix Include="StreamJsonRpc" />
     <NuGetPackageToIncludeInVsix Include="Newtonsoft.Json" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Threading" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Validation" />
```

Comment:
This seems very surprising to me. We're packaging a copy of a binary that won't be the version that's matching the real build?